definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "chrisamiller/homer"
```

Comment:
can you use a tagged version of the docker image?

Reply:
wouldn't be an awful idea to migrate this to an mgibio repo, so we can continue to use mine as a sandbox

Reply:
Is there a tagged version of you image @chrisamiller ?